### PR TITLE
[tool] Update CTSConverter to support 'TENSOR_QUANT8_SYMM_PER_CHANNEL' and 'TENSOR_QUANT8_ASYMM_SIGNED'

### PR DESCRIPTION
**Pull request by cuiyanx**
Signed-off-by: cuiyanx <yanx.cui@intel.com>

### CODE REVIEW

**Comment by cuiyanx** [comment]
@BruceDai PTAL

**Comment by cuiyanx** [comment]
@huningxin @BruceDai PTAL

**Comment by huningxin** [comment]
I am not familiar with this part. @BruceDai , please help review.

File: test/cts/tool/CTSConverter/src/cts_generator.py [hunk 1]
```diff
@@ -349,6 +351,19 @@ def DumpJSTest(model, example, js_fd):
             if t.type == "FLOAT16":
                 t.type = "FLOAT32"
 
+    '''
+    # select specifying type models
+    select_specifying_flag = False
+    if model.operations[0].optype == "CONV_2D" or model.operations[0].optype == "DEPTHWISE_CONV_2D":
```
Comment (BruceDai) [inline test/cts/tool/CTSConverter/src/cts_generator.py]: @cuiyanx Please update this code
```python
if model.operations[0].optype == "CONV_2D" or model.operations[0].optype == "DEPTHWISE_CONV_2D":
```
to extendable code
```python
if model.operations[0].optype in ["CONV_2D", "DEPTHWISE_CONV_2D"]:
```

File: test/cts/tool/CTSConverter/src/cts_generator.py [hunk 2]
```diff
@@ -349,6 +351,19 @@ def DumpJSTest(model, example, js_fd):
...
+        if model.operands[0].type.type == "TENSOR_QUANT8_ASYMM_SIGNED" and \
+           model.operands[1].type.type == "TENSOR_QUANT8_SYMM_PER_CHANNEL":
+            select_specifying_flag = True
+
+    if select_specifying_flag == False:
```
Comment (BruceDai) [inline test/cts/tool/CTSConverter/src/cts_generator.py]: @cuiyanx Don't compare boolean values to True or False using ==, please update this code
```python
if select_specifying_flag == False:
```
to
```python
if not select_specifying_flag:
```

File: test/cts/tool/CTSConverter/src/cts_generator.py [hunk 3]
```diff
@@ -349,6 +351,19 @@ def DumpJSTest(model, example, js_fd):
...
+        print ("    skip not select types: %s" %example.examplesName, file = sys.stderr)
```
Comment (BruceDai) [inline test/cts/tool/CTSConverter/src/cts_generator.py]: @cuiyanx Please add a blank space 
```python
%example.examplesName
```
to 
```python
% example.examplesName
```
and please update the others formatted strings code using '%'.

File: test/cts/tool/CTSConverter/src/cts_generator.py [hunk 4]
```diff
@@ -464,9 +479,12 @@ def DumpJSTest(model, example, js_fd):
                 return
 
     # set js test names
+    Configuration.example_count = Configuration.example_count + 1
```
Comment (BruceDai) [inline test/cts/tool/CTSConverter/src/cts_generator.py]: @cuiyanx Please update this code using '+=' augmented assignment .

**Comment by cuiyanx** [comment]
@BruceDai PTAL

**Comment by BruceDai** [comment]
LGTM, thanks @cuiyanx 
@ibelem Please help merge this PR, thanks

**Pull request merged**